### DISCUSSION TITLE
Team Collaboration

### DISCUSSION THREAD

**Discussion by aldhielman** [chatwoot/chatwoot#4443] [Q&A]
Supposed I have chat from customer on **Inbox A**, but unfortunately i can handle those chat so i assign those conversation to **Team ZZZ**, but the member of the **Team ZZZ** have no access on **Inbox A**. It's make my assignmet to Team ZZZ cannot "delivered" to Team ZZZ (either the member) and they cannot handle those conversation.

Did I miss something ? 
How make it possible to Team ZZZ participate on those conversation ?

Update : it doesn't work while using mention on private note

**Comment by sojan-official** [comment]
lets track this over https://github.com/chatwoot/chatwoot/issues/4444
